query/analyze/structure.feature

```diff
@@ -0,0 +1,241 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#noinspection CucumberUndefinedStep
+Feature: Basic Analyze queries
```

Comment:
This should be "Query structure"

Then that hints that you'd want another file that's "Query annotations" that tests the annotations part right?